### DISCUSSION TITLE
Boxart

### DISCUSSION THREAD

**Discussion by null-character** [OnionUI/Onion#793] [Ideas]
Is it possible to hide the box art from the game list but show it on the game detail screen accessed to the right?

With how small the screen is on the mini and mini+ the box art doesn't fit well in the main game list at all.

**Comment by Aemiii91** [comment]
Yes, you can do this by replacing `skin/preview-bg.png` in your current theme with a 1x1px transparent PNG.

**Comment by null-character** [comment]
Awesome thanks.
